woocommerce/woocommerce-ios
# Products tab: fix old Product Detail still on the screen when switching stores

Fixes #1642 

## Changes

- Products tab: popped to navigation root view controller when the store changes (e.g. after switching stores)

## Testing

Prerequisite: the account has at least 2 stores with some Products
- Launch the app
- Go to Products tab
- Tap on a Product
- Go to "My store" tab
- Tap on Settings and then switch to another store
- Go to Products tab --> the Product list of the new store should be shown, instead of the old Product details

Update release notes:
